packages/renovate-config-packages/package.json

```diff
@@ -59,7 +59,8 @@
       "extends": [
         "packages:eslint",
         "packages:stylelint",
-        "packages:tslint"
+        "packages:tslint",
+        "monorepo:typescript-eslint"
```

Comment:
We need to double check what this combination expands to. Sometimes multiple nested presets can conflict with each other (e.g. by introducing unintended "AND" logic instead of "OR") and result in nothing matching.

Reply:
Thanks for the input @rarkins. Closing this for now.

Reply:
It seems that using `packagePatterns` instead of `sourceUrlPrefixes` in `monorepo:typescript-eslint` will be a possible solution.